src/content/app/tools/blast/components/listed-blast-submission/ListedBlastSubmission.test.tsx

```diff
@@ -0,0 +1,235 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { Provider } from 'react-redux';
+import { MemoryRouter } from 'react-router';
+import { configureStore, combineReducers } from '@reduxjs/toolkit';
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import * as blastStorageService from 'src/content/app/tools/blast/services/blastStorageService';
+
+import ListedBlastSubmission, {
+  type Props
```

Comment:
Can we create a meaningful alias for this?

Reply:
✅